src/app/core/models/github/github-issue-filter.model.ts

```diff
@@ -17,7 +17,7 @@ export interface RestGithubIssueFilterData {
 }
 /**
  * A filter to filter out the issues to retrieve from github.
- * Ref: https://developer.github.com/v3/issues/#parameters-3
+ * Ref: https://docs.github.com/en/rest/issues/issues#list-repository-issues
```

Comment:
Wrong link, it should be a reference to IssueFilter

Reply:
appears to be in the right ballpark - https://web.archive.org/web/20200917232046/https://developer.github.com/v3/issues/#parameters-3

but i've updated the link to point to the list of params found in a issue filter to better reflect this.